go.mod

```diff
@@ -250,6 +249,7 @@ require (
 	github.com/jonboulle/clockwork v0.2.2 // indirect
 	github.com/kevinburke/ssh_config v1.2.0 // indirect
 	github.com/klauspost/compress v1.16.5 // indirect
+	github.com/kolide/kit v0.0.0-20221107170827-fb85e3d59eab // indirect
```

Comment:
Any idea who is using this indirectly?

Reply:
I tried the following but gave me no answer:
```
go mod why github.com/kolide/kit
# github.com/kolide/kit
(main module does not need package github.com/kolide/kit)
```
My guess is `github.com/kolide/launcher` (which I can see being imported on such repo).

Reply:
Ok. Looks like we are using launcher in `launcher.go`